src/lyapunovs.jl

```diff
@@ -42,11 +42,9 @@ function lyapunovs(ds::DiscreteDS, N::Real; Ttr::Int= 100)
     J = jac(u)
     K = J*Q
 
-    F = qrfact(K)
-    Q .= F[:Q]
-    A_mul_signB!(Q, F[:R]) #ensure QR gives positive diagonal to R
```

Comment:
Not necessary, ```abs(R[i,i])``` makes sure they are positive. As you can see, this also yield exactly the same Lyapunov exponents. Please correct me, if I overlooked something.